packages/core/src/utils/styleUtils/stylesUtils.ts

```diff
@@ -98,7 +107,26 @@ export const buildCfStyles = ({
     objectFit: cfImageOptions?.objectFit,
     objectPosition: cfImageOptions?.objectPosition,
   };
+  const filteredRules = Object.fromEntries(
+    Object.entries(translatedRules).filter(([, value]) => value !== undefined),
+  );
+  return filteredRules;
 };
+
+export const buildCfStylesWithSpecialCasing = (
```

Comment:
this name is too cryptic. If I had to choose between `buildCfStyles` and `buildCfStylesWithSpecialCasing` I wouldn't know which one to call.

Can we refactor it to be called `adjustStructureComponent`, which takes teh `cfStyles` as input (so `buildCfStyles` is called outside) and then returns adjusted styles as output?

Reply:
I don't know why we're running this check only in some cases which is why I'm not merging it into `buildCfStyles` for now. I like your first recommendation - will use that name and remove `buildCfStyles`

Reply:
I think I finally understood that this is only meant to be used for editor mode where empty containers should have a min height. I'm not exactly sure if we're already doing this accidentally for preview with the currently published SDK version.